installers/binstall/install.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+#
+# Licensed under the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+# This is just a little script that can be downloaded from the internet to
+# install rover. It just does platform detection, downloads the installer
+# and runs it.
+
+set -u
+
+BINARY_DOWNLOAD_PREFIX="https://github.com/apollographql/rover/releases/download/$VERSION"
```

Comment:
@EverlastingBugstopper the biggest missing piece from this (afaict) is the abilty to download "latest" with this script. Am I missing that somewhere in the setup here?

Reply:
no, you're not missing that. the way `wasm-pack` does this is in their documentation build step, they actually replace the `$VERSION` string with the version that the documentation was built with.

i'm not sure what our installer page will look like or if we want to just always default to the latest published version by doing an API call to GitHub or if we want it to be pseudo-hard-coded or not. I figured since this is just for gravity right now we could just have the first install command look like this and then we can figure out the way we do this _actually_ a bit later.

`curl -sSL https://raw.githubusercontent.com/apollographql/rover/v0.0.1/installers/binstall/install.sh | VERSION=v0.0.1 sh`

happy to either file an issue for this or we can talk about how to address it now!

Reply:
Ah, one trick we used for the old installer was looking at release/latest with a HEAD request which gets you the url (with a version number) of the lastest release